docs/build/training.md

```diff
@@ -1,40 +1,76 @@
 ---
 title: Build for training
 parent: Build ONNX Runtime
-description: Learn how to build ONNX Runtime for training from source for different hardware targets
+description: Learn how to build ONNX Runtime for training from source for different scenarios and hardware targets
 nav_order: 2
 redirect_from: /docs/how-to/build/training
 ---
 
-# Build ONNX Runtime for training
+# Build for On-Device Training
 {: .no_toc }
 
-## Contents
-{: .no_toc }
+## Prerequisites
 
-* TOC placeholder
-{:toc}
+- Python 3.x
+- CMake
 
-## CPU
+## Build Instructions for the Training Phase
 
-### Build Instructions
-{: .no_toc }
+1. Clone the repository
 
-To build ORT with training support add `--enable_training` build instruction.
+    ```bash
+    git clone --recursive https://github.com/Microsoft/onnxruntime.git
+    cd onnxruntime
+    ```
 
-All other build options are the same for inferencing as they are for training.
+2. Build onnxruntime for `On-Device Training`
```

Comment:
nit, when we specify onnxruntime, maybe we should use a consistent style calling it, I see there is "ONNX Runtime" and "onnxruntime", I would prefer the former one. :)